app/models/discount_code.py

```diff
@@ -44,9 +43,7 @@ def __init__(self,
                  created_at=None,
                  used_for=None,
                  event_id=None,
-                 tickets=None,
-                 user_id=None,
-                 deleted_at=None):
```

Comment:
why remove the `deleted_at` ?

Reply:
FYI, `deleted_at` is a column that is added to every model/schema that needs soft deletion support. discount code is one of them.